packages_rs/nextclade/src/align/params.rs

```diff
@@ -143,7 +147,8 @@ impl Default for AlignPairwiseParams {
       penalty_gap_open_out_of_frame: 8,
       penalty_mismatch: 1,
       score_match: 3,
-      max_indel: 400,        // to be replaced
+      max_band_area: 500_000_000,
```

Comment:
Default value is here. 

The usual precedence order apply:

 - CLI param (or Web param if we ever have it in the UI) has the highest priority
 - Then dataset defaults from `virus_properties.json`/`pathogen.json`
 - If none of that, than this hardcoded value